code-samples/creating-your-bot/event-handling/index.js

```diff
@@ -1,9 +1,17 @@
 const fs = require("node:fs");
 const path = require("node:path");
-const { Client, GatewayIntentBits } = require("discord.js");
+const { Client, GatewayIntentBits, Partials } = require("discord.js");
 const { token } = require("./config.json");
 
-const client = new Client({ intents: [GatewayIntentBits.Guilds] });
+const client = new Client({ 
+  intents: [
+    GatewayIntentBits.Guilds,
+    GatewayIntentBits.GuildMembers,
+  ],
+  partials: [
+    Partials.GuildMember,
+  ],
```

Comment:
Do we need to account for Partial events or Intents in [guildMemberUpdate.js](https://github.com/Slyracoon23/DiscordBot-Discourse-Sync/pull/1/files?diff=split&w=0#diff-a7fde68ca87ade890c17e2ecabb6fe485a62248a695f434dffc8145f6ede20c0) ?

Reply:
We needed to add partial events and intents in otherwise the bot won't recognized uncached members.